packages/manager/modules/pci/src/projects/project/additional-ips/failover-ips/failover-ips.html

```diff
@@ -0,0 +1,92 @@
+<div data-ui-view>
+    <oui-datagrid
+        rows="$ctrl.failoverIps"
+        criteria="$ctrl.criteria"
+        on-criteria-change="$ctrl.onListParamChange($criteria)"
+    >
+        <oui-datagrid-topbar>
+            <oui-action-menu
+                data-text="{{:: 'pci_additional_ips_actions' | translate }}"
+            >
+                <oui-action-menu-item
+                    data-on-click="$ctrl.goToImportFailoverIps()"
+                >
+                    <span
+                        data-translate="pci_additional_ips_import_failover_ip"
+                    ></span>
+                </oui-action-menu-item>
+                <oui-action-menu-item
+                    data-on-click="$ctrl.goToAdditionalIpOrderPage()"
+                >
+                    <span data-translate="pci_additional_ips_order"></span>
+                </oui-action-menu-item>
+            </oui-action-menu>
+        </oui-datagrid-topbar>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_failover_ip_title' | translate"
+            data-property="ip"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_ip_block' | translate"
+            data-property="block"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_geoloc' | translate"
+            data-property="geoloc"
+            data-type="string"
+        >
+            <span
+                data-ng-bind="'pci_additional_ips_country_' + $row.geoloc | translate"
+            ></span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_id' | translate"
+            data-property="id"
+            data-type="string"
+        >
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_additional_ips_routedTo' | translate"
+            data-property="instance.name"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+            <a
+                data-ng-if="$row.instance"
+                data-ui-sref="pci.projects.project.instances.instance({projectId : $ctrl.projectId, instanceId: $row.instance.id})"
```

Comment:
**question:** Couldn't be replaced by a `ngHref`? (as we tend to avoid using `ui-sref` directive).

Reply:
Done.